java-frontend/src/main/java/org/sonar/java/bytecode/visitor/ResourceMapping.java

```diff
@@ -19,62 +19,24 @@
  */
 package org.sonar.java.bytecode.visitor;
 
-import com.google.common.collect.ArrayListMultimap;
-import com.google.common.collect.ImmutableSet;
-import com.google.common.collect.ImmutableSortedSet;
-import com.google.common.collect.Multimap;
-import org.sonar.api.design.Dependency;
-import org.sonar.api.resources.Directory;
-import org.sonar.api.resources.File;
-import org.sonar.api.resources.Resource;
-
-import java.util.Collection;
-import java.util.Comparator;
 import java.util.HashMap;
 import java.util.Map;
-import java.util.Set;
 
 public class ResourceMapping {
```

Comment:
This whole class can probably be removed now, as it is now just a container of a Map!